### PR TITLE
Issue 1284 3

### PR DESCRIPTION
**Pull request by Naupio**
## What problem does this PR solve?
fix erc721_udts with invalid holders count return
#### Test
- unit test
#### Task
 - none

### CODE REVIEW

File: lib/godwoken_explorer/graphql/resolovers/udt.ex [hunk 1]
```diff
@@ -489,7 +489,7 @@ defmodule GodwokenExplorer.Graphql.Resolvers.UDT do
       from(cu in CurrentUDTBalance)
       |> where([cu], cu.address_hash != ^minted_burn_address_hash)
       |> order_by([cu], desc: cu.block_number, desc: cu.id)
-      |> distinct([cu], cu.address_hash)
+      |> distinct([cu], [cu.address_hash, cu.token_contract_address_hash])
```
Comment (zmcNotafraid) [inline lib/godwoken_explorer/graphql/resolovers/udt.ex:492]: CurrentUDTBalance 会存在一个用户有多条相同的udt ？

**Review by zmcNotafraid — CHANGES_REQUESTED** [review]

**Comment by FrederLu** [comment]
This data shows that holders is 32, but it is actually 1.
https://godwoken-explorer-ui-development.vercel.app/nft-collection/0x821543e1e3923bb6c7efa6e5ebd5f83da3b25cd8?tab=holders
image
image

**Comment by Naupio** [comment]
> This data shows that holders is 32, but it is actually 1. https://godwoken-explorer-ui-development.vercel.app/nft-collection/0x821543e1e3923bb6c7efa6e5ebd5f83da3b25cd8?tab=holders image image

fixed https://github.com/Magickbase/godwoken_explorer/pull/1361/commits/242da748f7d1a4751ec895b3cabb3e62f837c33b

File: lib/godwoken_explorer/graphql/resolovers/udt.ex (hunk 1 above)
Comment (Naupio) [reply lib/godwoken_explorer/graphql/resolovers/udt.ex]: 用 CurrentUDTBalance 算 count 的时候, erc721 不同 token id 的 value 意义不一样, 所以排序之后 distinct, 不过这部分现在改了 ,erc721 holder count 用 新表 erc721_token 去算了

**Review by zmcNotafraid — APPROVED** [review]

**Comment by FrederLu** [comment]
The amount traded in holders was found not to be reduced.
https://godwoken-explorer-ui-development.vercel.app/zh-CN/nft-collection/0x64642a6025439b0d550a862ff9086a61e54d9f88?tab=holders
image

**Comment by FrederLu** [comment]
https://godwoken-explorer-ui-development.vercel.app/zh-CN/multi-token-collection/0xeeb92d6b3999abdc8cc0226069d88c07f241fd4c?tab=holders
image

**Review by FrederLu — APPROVED** [review]

**Pull request merged**